modules/rpc/yarpc.go

```diff
@@ -126,10 +126,12 @@ func (c *dispatcherController) addDefaultMiddleware(host service.Host) {
 	cfg := yarpcConfig{
 		inboundMiddleware: []middleware.UnaryInbound{
 			contextInboundMiddleware{host},
+			panicInboundMiddleware{},
```

Comment:
I think panic middleware should be the very first in the slice, to catch all possible issues.